Libraries/LibWeb/HTML/Navigable.h

```diff
@@ -188,6 +189,11 @@ class Navigable : public JS::Cell
     Web::EventHandler& event_handler() { return m_event_handler; }
     Web::EventHandler const& event_handler() const { return m_event_handler; }
 
+    // https://drafts.csswg.org/css-view-transitions-1/#snapshot-containing-block
+    CSSPixelRect snapshot_containing_block();
+    // https://drafts.csswg.org/css-view-transitions-1/#snapshot-containing-block-size
+    Tuple<CSSPixels, CSSPixels> snapshot_containing_block_size();
```

Comment:
Have to page in @alimpfard for this, but I believe `AK::Tuple` is meant to be used for metaprogramming only - use a struct for this instead.